Symfony/CS/Tests/Fixer/Contrib/NoUselessReturnFixerTest.php

```diff
@@ -34,6 +34,44 @@ public function provideFixCases()
         return array(
             array(
                 '<?php
+                    function bar($baz)
+                    {
+                        if ($baz)
+                            return $this->baz();
+                        else
```

Comment:
>say, empty else blocks

the trick is of course the `else`-block not being empty, just useless since it is the last block and returns as PHP does on default (for now), 
having the AST would be nice (long wait though ;)  )